Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/view/TestResultSectionView.kt

```diff
@@ -26,10 +26,11 @@ constructor(
     attrs: AttributeSet? = null,
     defStyleAttr: Int = 0
 ) : ConstraintLayout(context, attrs, defStyleAttr) {
-    private val binding = ViewTestResultSectionBinding.bind(this)
+    private val binding: ViewTestResultSectionBinding
 
     init {
         inflate(context, R.layout.view_test_result_section, this)
+        binding = ViewTestResultSectionBinding.bind(this)
```

Comment:
could be replaced by just calling

`binding = ViewTestResultSectionBinding.inflate(this)`